includes/class-pwa.php

```diff
@@ -16,6 +16,13 @@
  */
 class PWA {
 
+	/**
```

Comment:
@rgomezp this commit adds the service worker scope you recommended.

Reply:
@jeffersonrabb Looks good to me!